codes/quantum/qubits/movassagh_ouyang

```diff
@@ -0,0 +1,86 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: movassagh_ouyang
+physical: qubits
+logical: qubits
+
+
+name: 'Movassagh-Ouyang Codes'
+
+introduced: '\cite{arXiv:2012.01453}'
+
+description: 'This is a family of codes derived via an algorithm that takes as input \textit{any} classical code and outputs a quantum code. The algorithm is probabalistic but suceeds with high probability (almost surely if the classical code is random). This family of codes strictly generalizes CSS codes (because CSS codes come only from linear or self orthogonal classical codes). This family of codes is not a subset of stabilizer codes nor are stabilizer codes a subset of this family. This family of codes can be shown to be realized as a subspace of the ground space of a local Hamiltonian (and so they can be considered quantum LDPC codes).
```

Comment:
Is the Ham geometrically local or just "CS" local?

Reply:
What is the logical dimension here, just 2? Is this what you mean by your comment in "notes" about this being extendable to qudits? Please clarify.